### PR TITLE
Luke: tms patch

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

File: ion/services/dm/transformation/transform_management_service.py [hunk 1]
```diff
@@ -26,6 +22,31 @@ def __init__(self):
 
         self.serializer = IonObjectSerializer()
 
+    def on_start(self):
+        super(TransformManagementService,self).on_start()
+        restart_flag = self.CFG.get_safe('process.restart', False)
```
Comment (dstuebe) [inline ion/services/dm/transformation/transform_management_service.py]: Change this to 'service.transform_management.restart'

File: ion/services/dm/transformation/transform_management_service.py [hunk 2]
```diff
@@ -26,6 +22,31 @@ def __init__(self):
...
+
+        if len(proc_def_ids) < 1:
+            log.warning('Transform did not have a correct process definition.')
+            return
+
+        self.clients.process_dispatcher.schedule_process(
+            process_definition_id=proc_def_ids[0],
+            configuration=configuration
+        )
+
```
Comment (dstuebe) [inline ion/services/dm/transformation/transform_management_service.py:46]: Update the pid in the transform resource.

File: ion/services/dm/transformation/transform_management_service.py (hunk 1 above)
Comment (lukecampbell) [inline ion/services/dm/transformation/transform_management_service.py]: Done

File: ion/services/dm/transformation/transform_management_service.py (hunk 2 above)
Comment (lukecampbell) [inline ion/services/dm/transformation/transform_management_service.py:46]: Done

**Pull request merged**